src/helm/benchmark/contamination/compute_contamination_metrics.py

```diff
@@ -0,0 +1,248 @@
+import json
+import argparse
+import os
+import glob
+
+from typing import List, Tuple, Set
+from nltk import ngrams
+from typing import Dict, Optional, DefaultDict
+from collections import defaultdict
+
+from helm.benchmark.contamination.light_scenario import LightInstance, LightScenario
+from helm.benchmark.contamination.light_tokenizer import LightTokenizer, DefaultTokenizer
+from helm.benchmark.contamination.document_reading_processor import DocumentReadingProcessor
+from helm.benchmark.contamination.contamination_stats import ContaminationStats, PART_INPUT, PART_REF
+
+
+# The n values of the ngrams to be computed
+N_VALUES: List[int] = [5, 9, 13]  # TODO: Pick the N values
+
+
+def load_scenarios_from_jsonl(path: str) -> List[LightScenario]:
+    """
+    Create a list of light scenarios from a jsonl file, where each json represents a LightScenario object.
+
+    Input file format:
+
+    Instance JSON 1
+    Instance JSON 2
+    Instance JSON 3
+    ...
+
+    Each line is a json and each json looks like:
+    {
+        "scenario_spec": {
+            "name": "SCENARIO_NAME",
```

Comment:
Specify what name this is?  The class_name with the args?  Or the run spec name?